db/jobs.go

```diff
@@ -86,17 +91,25 @@ func (f *JobFilter) Query() (string, []interface{}) {
 	}
 
 	return `
+	WITH most_recent_job_task_at AS (
+		SELECT job_uuid,  max(requested_at) AS requested_at FROM tasks group by job_uuid
+	),
+	most_recent_job_task AS (
+		SELECT t.started_at, t.status, t.job_uuid, t.uuid FROM tasks t, most_recent_job_task_at mr WHERE t.job_uuid = mr.job_uuid AND t.requested_at = mr.requested_at
+	)
```

Comment:
This won't work for MySQL, so maybe we should just omit the status check and MySQL implementations won't get monitoring?